packages/js-dpp/lib/identity/stateTransition/IdentityUpdateTransition/validation/state/validatePublicKeysState.js

```diff
@@ -21,10 +21,8 @@ const MaxIdentityPublicKeyLimitReachedError = require(
    * @param {RawIdentityPublicKey[]} rawPublicKeys
    *
    * @return {ValidationResult}
-   */
-function validatePublicKeysState(rawPublicKeys) {
+   */ function validatePublicKeysState(rawPublicKeys) {
```

Comment:
I have a feeling that terminating the comment on the same line as function starts was intentional 😅